tests/testthat/test_ExtractData_Soils.R

```diff
@@ -18,23 +19,26 @@ do_skip <- c(
   # mimmics skip_on_appveyor():
   identical(tolower(Sys.getenv("APPVEYOR")), "true"))
 
-suppressWarnings(is_online <-
-                   !inherits(try(close(url(getOption("repos"), open = "r")), silent = TRUE),
-                             "try-error"))
 
-
-if (!any(do_skip) && is_online) {
+if (!any(do_skip)) {
   # =============================================================================
   # Tests designed to test the underlining structures created
   # from soil extraction functions.
   # =============================================================================
   
-  # set stage manually so that future changes won't cause the test to fail ======
   # setup file paths
   fnames_in <- environment()
   fnames_in$fslayers <- file.path("/home/natemccauslin/Desktop/Dryland Ecology/rSFSW2/tests/test_data/TestPrj4/1_Input/SWRuns_InputData_SoilLayers_v9.csv")
```

Comment:
Why absolute paths for `fslayers` and `fsoils` specific to your computer? The folder `TestPrj4` is part of this package source, thus, it will always be the same and you should use a relative path, e.g., see example in `test_projects.R` which uses `dir_tests <- file.path("..", "test_data", "TestPrj4")`